.github/workflows/build.yaml

```diff
@@ -79,6 +80,6 @@ jobs:
         run: tar -xvf ttm_${{ matrix.arch }}-${{ matrix.config}}.tar
       - name: Build tt-metal tests
         run: |
-          rm -f build/CMakeCache.txt
-          cmake -B build -G Ninja
-          cmake --build build --target tests -- -j`nproc`
+          rm -f build/CMakeCache.txt && find build -name 'cmake_pch.hxx.pch' -exec touch {} \;
```

Comment:
Is this to force PCH recompile? Why is this needed if they don't change between the lib and test build?

Reply:
It's to tell clang that we don't need to recompile PCH, otherwise it'll complain it's outdated even though nothing changed :(